src/visualizer/controller.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+    const {
+        getListOfData,
+        getData
+    } = window.comm;
+    const {
+        createGroupInfoElement,
+        createAverageValueTable
+    } = window.detailsElement;
+    const {
+        createGroupSvg
+    } = window.svg;
+
+    getListOfData().then(responds => {
+        console.log(responds);
+        createButtons(responds);
+    });
+
+    /**
+     * @summary Creates a button for each file in the data folder
+     * @param {Array} array this is an array with the names of the files in the data folder
+     */
+    function createButtons(array) {
+        for (let i = 0; i < array.length; i++) {
```

Comment:
Brug et for of

Reply:
Done, faktisk inden dit review XD, glemte at commit